Makefile

```
@@ -483,6 +483,10 @@ build-capi-headless-all: capi-setup
 	RUSTFLAGS="${RUSTFLAGS}" cargo build --manifest-path lib/c-api/Cargo.toml --release \
 		--no-default-features --features universal,dylib,staticlib,wasi
 
+build-capi-headless-ios: capi-setup
```

```suggestion
build-capi-headless-dylib-ios: capi-setup
```

Since it uses the same features than `build-capi-headless-dylib`, I think it's important to get `dylib` in the target/recipe name. Thoughts?

---

Yes that's a good idea!